packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -258,9 +307,16 @@ class Auth {
         this.set('customer_type', isGuest ? 'guest' : 'registered')
 
         const refreshTokenKey = isGuest ? 'refresh_token_guest' : 'refresh_token_registered'
+        const refreshTokenCopyKey = isGuest
+            ? 'refresh_token_guest_copy'
+            : 'refresh_token_registered_copy'
+
         this.set(refreshTokenKey, res.refresh_token, {
             expires: this.REFRESH_TOKEN_EXPIRATION_DAYS
         })
+        this.set(refreshTokenCopyKey, res.refresh_token, {
```

Comment:
So we are always storing a refresh_token and a copy of it for every app consuming this library. Do you think it is better if we have a flag to indicate if an app is hybrid, and only store this key when it is true? 
Nit: the name `refreshTokenCopyKey` is a bit vague and too generic here, maybe we can call it `refreshTokenPwaKit` to indicate this is for pwa-kit side? 🤔

Reply:
One thing about the `refreshTokenCopyKey` that may be confusing is that in an actual running PWA, what someone who opens DevTools will see is a refresh token (ie. cc-nx-g) stored in cookie store (the main one that we use) and a copy with the exact same name in local store.

This might cause some confusion for someone who is debugging so we'll want to document why we've got this copy somewhere.

Reply:
Again, We decided on keeping the default state as hybrid. So we're not adding flags for isHybrid. I'll add a comment to explain the copy though.